features/environment.py

```diff
@@ -5,8 +5,8 @@
 import json
 from selenium import webdriver
 
-WAIT_SECONDS = int(os.getenv('WAIT_SECONDS', '60'))
-BASE_URL = os.getenv('BASE_URL', 'http://localhost:5000')
+WAIT_SECONDS = int(getenv('WAIT_SECONDS', '60'))
+BASE_URL = getenv('BASE_URL', 'https://nyu-customer-service-fall2101-dev.us-south.cf.appdomain.cloud')
```

Comment:
@YongchangLiu98 we can't change it like this because then the BDD tests do not run locally. We need to be able to run them both locally and in the cloud, which is why it's conditionally set at the moment. I added a BASE_URL variable to the integration stage of the ibm cloud toolchain hoping to set the BASE_URL correctly in the cloud using lines 11 and 12, but let me know if it's not working